public-interface/dashboard/public/js/controllers/charts/charts.js

```diff
@@ -497,13 +497,26 @@ iotController.controller('ChartCtrl', function( $scope,
             })[0];
 
             if (oldSerie) {
-                var newPoints = serie.points.filter(function(point) {
+                var newPoints = oldSerie.points.length > 0 ? serie.points.filter(function(point) {
                     // we only want to add points newer than those already in serie
                     // and serie is ordered, so we can just compare to last point
                     return (point.ts > oldSerie.points[oldSerie.points.length - 1].ts);
-                });
+                }) : [];
+                if(newPoints.length > 0){
```

Comment:
@gabriellewp This looks like code from a previous PR. You probably need to rebase your forked repo